extensions/yaml_support/labgraph_monitor/server/lg_monitor_server.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+# Copyright 2004-present Facebook. All Rights Reserve
+
+import labgraph as lg
+from typing import Tuple
+from labgraph.websockets.ws_server.ws_api_node_server import (
+    WSAPIServerConfig,
+    WSAPIServerNode
+)
+from .serializer_node import SerializerConfig, Serializer
+from ..aliases.aliases import SerializedGraph
+from .enums.enums import ENUMS
+
+APP_ID = 'LABGRAPH_MONITOR'
+WS_SERVER = ENUMS.WS_SERVER
+STREAM = ENUMS.STREAM
+DEFAULT_IP = WS_SERVER.DEFAULT_IP
+DEFAULT_PORT = WS_SERVER.DEFAULT_PORT
+DEFAULT_API_VERSION = WS_SERVER.DEFAULT_API_VERSION
+SAMPLE_RATE = 5
+
+
+def run_server(data: SerializedGraph) -> None:
+    """
+    A function that creates a Websocket server graph.
+    The server graph streams the lagraph topology to the clients
+    """
+    class WSSenderNode(lg.Graph):
+        SERIALIZER: Serializer
+        WS_SERVER_NODE: WSAPIServerNode
+
+        def setup(self) -> None:
+            wsapi_server_config = WSAPIServerConfig(
+                app_id=APP_ID,
+                ip=WS_SERVER.DEFAULT_IP,
+                port=ENUMS.WS_SERVER.DEFAULT_PORT,
+                api_version=ENUMS.WS_SERVER.DEFAULT_API_VERSION,
+                num_messages=-1,
+                enums=ENUMS(),
+                sample_rate=SAMPLE_RATE,
+            )
+            self.SERIALIZER.configure(
+                SerializerConfig(
+                    data=data,
+                    sample_rate=SAMPLE_RATE,
+                    stream_name=STREAM.LABGRAPH_MONITOR,
+                    stream_id=STREAM.LABGRAPH_MONITOR_ID
+                )
+            )
```

Comment:
Just to be sure @bennaaym, does this `SERIALIZER` publish:

- The current state of the graph (such as it is running the `Demo` graph `lg.run(Demo)`  as well as the `WSSenderNode` graph `lg.run(WSSenderNode)`)
- Or does it just publish the data that it received from `serialize_graph()` without running `lg.run(Demo)`

I think there might be a small issue with the fact that it is not running the `Demo` graph while running the `WSSenderNode`, which makes it impossible to fetch real-time data right now

Reply:
do you know if there is a good way of running the `Demo` graph as well as the `WSSenderNode`?

Reply:
Hey, @dtemir I was assuming that the graph is running in a different process. using,  lg.run will block the main thread till the execution end. 

a way around that might be to use a threading library to run the both  graphs: 

```
import labgraph as lg
from labgraph.examples.simple_viz import Demo 
from extensions.yaml_support.labgraph_monitor.generate_lg_monitor.generate_lg_monitor import generate_labgraph_monitor
from  threading import Thread


def run_graph_in_new_thread(graph: lg.Graph):
    runner = lg.ParallelRunner(graph=graph)
    runner.run()

if __name__ == '__main__':
    graph = Demo()
    thread = Thread(target=run_graph_in_new_thread, args=(graph, ))
    thread.start()
    generate_labgraph_monitor(graph)
    thread.join()
```

Reply:
thanks a lot @bennaaym! This is very-very useful. Now I gotta find a way to move the messages between two graphs